app/javascript/elm/tests/MobileSessionsFiltersTests.elm

```diff
@@ -133,53 +101,21 @@ tagsArea =
 profileNamesArea : Test
 profileNamesArea =
     describe "Profile names tests: "
-        [ describe "given a search field"
-            [ fuzz string "when user types, UpdateProfileSearch is triggered with the input" <|
-                \inputValue ->
-                    defaultModel
-                        |> view
-                        |> Query.fromHtml
-                        |> Query.find [ Slc.id "test-profiles" ]
-                        |> Query.find [ Slc.tag "input" ]
-                        |> Event.simulate (Event.input inputValue)
-                        |> Event.expect (UpdateProfileSearch inputValue)
-            , fuzz string "when UpdateProfileSearch is triggered the input is displayed in the search field" <|
-                \inputValue ->
-                    defaultModel
-                        |> update (UpdateProfileSearch inputValue)
-                        |> Tuple.first
-                        |> view
-                        |> Query.fromHtml
-                        |> Query.find [ Slc.id "test-profiles" ]
-                        |> Query.find [ Slc.tag "input" ]
-                        |> Query.has [ Slc.attribute <| Attr.value inputValue ]
-            ]
-        , describe "when AddProfile is triggered"
-            [ fuzz string "the search field value is reset" <|
-                \activityValue ->
-                    { defaultModel | profiles = Labels.updateCandidate defaultModel.profiles "some string" }
-                        |> update (AddProfile activityValue)
-                        |> Tuple.first
-                        |> view
-                        |> Query.fromHtml
-                        |> Query.find [ Slc.id "test-profiles" ]
-                        |> Query.find [ Slc.tag "input" ]
-                        |> Query.has [ Slc.attribute <| Attr.value "" ]
-            , fuzz string "new profile label is created" <|
+        [ describe "when AddProfile is triggered"
+            [ fuzz string "new profile label is created" <|
                 \activityValue ->
                     defaultModel
                         |> update (AddProfile activityValue)
                         |> Tuple.first
                         |> view
                         |> Query.fromHtml
                         |> Query.has [ Slc.text activityValue ]
-            , fuzz string "new profile label has a button" <|
-                \profileName ->
-                    { defaultModel | profiles = Labels.fromList [ profileName ] }
+            , test "new profile label has a button" <|
```

Comment:
Was the `fuzz` a problem on this one?

Reply:
Yes, cause it sometimes produces an empty string. I think at some point I will create a custom `fuzz` that produces strings that I need and I'll revisit these tests.